src/library/dlgtagfetcher.cpp

```diff
@@ -112,6 +130,22 @@ void DlgTagFetcher::loadTrack(const TrackPointer& track) {
     updateStack();
 }
 
+void DlgTagFetcher::loadTrack(const TrackPointer& track) {
+    VERIFY_OR_DEBUG_ASSERT(!m_pTrackModel) {
+        return;
+    }
+    changeTrack(track);
```

Comment:
This function is redundant with `changeTrack`. Pick one name and delete the other function.